core/storage/job/gae_models.py

```diff
@@ -86,6 +93,15 @@ def is_cancelable(self):
 
     @classmethod
     def get_recent_jobs(cls, limit, recency_msec):
+        """Gets no more than a number of jobs within a given recency.
```

Comment:
Be more concrete: "Gets at most `limit` jobs that ..."

And also, it's not clear what "within a given recency" means -- can you elaborate? Feel free to spill over to the next line if you need to.